packages/teams-js/test/private/files.spec.ts

```diff
@@ -891,6 +1008,43 @@ describe('files', () => {
       );
     });
 
+    it('should not allow calls with null request', async () => {
+      await utils.initializeWithContext('content');
+      expect(() => files.uploadCloudStorageProviderFile(null, emptyCallback)).toThrowError(
+        '[files.uploadCloudStorageProviderFile] 3P cloud storage provider request content details are missing',
+      );
+    });
+
+    it('should not allow calls with null request content', async () => {
+      await utils.initializeWithContext('content');
+      expect(() => files.uploadCloudStorageProviderFile(uploadFileRequestWithNullContent, emptyCallback)).toThrowError(
+        '[files.uploadCloudStorageProviderFile] 3P cloud storage provider request content details are missing',
+      );
+    });
+
+    it('should not allow calls with empty itemList in request content', async () => {
+      await utils.initializeWithContext('content');
+      expect(() =>
+        files.uploadCloudStorageProviderFile(uploadFileRequestWithEmptyItemList, emptyCallback),
+      ).toThrowError(
+        '[files.uploadCloudStorageProviderFile] 3P cloud storage provider request content details are missing',
+      );
+    });
+
+    it('should not allow calls with null destinationFolder request content', async () => {
+      await utils.initializeWithContext('content');
+      expect(() =>
+        files.uploadCloudStorageProviderFile(uploadFileRequestWithNullDestinationFolder, emptyCallback),
+      ).toThrowError('[files.uploadCloudStorageProviderFile] Invalid destination folder details');
+    });
+
+    it('should not allow upload calls for file as destination folder in request content', async () => {
```

Comment:
I don't think you have a test for the CloudStorageProvider.Sharepoint case

Reply:
Yes i didn't. Added now.